aten/src/ATen/TensorIterator.cpp

```diff
@@ -572,20 +572,20 @@ DimVector TensorIteratorBase::invert_perm(IntArrayRef input) const {
 }
 
 void TensorIteratorBase::allocate_or_resize_outputs() {
+  // check if permutation is just an inverted order
+  bool inverted = true;
```

Comment:
note to self, we should probably refactor this into a private method as some point.